js/main.js

```diff
@@ -0,0 +1,49 @@
+const app = new Vue({
+  el: '#app',
+  data: {
+    newItem: '',
+    todos: []
+  },
+  mounted: function() {
```

Comment:
shorthand とそうでないものがプログラム中で混在しているので shorthand に寄せましょう。

```suggestion
  mounted () {
```

Reply:
修正しました。